test/e2e/data/shared/v1beta1-caren/metadata.yaml

```diff
@@ -43,3 +43,6 @@ releaseSeries:
   - contract: v1beta1
     major: 0
     minor: 18
+  - contract: v1beta1
+    major: 0
+    minor: 19
```

Comment:
If we're not bumping to v0.19.99 in caren.yaml then we don't need this here.

Reply:
Removed.